doc/nrf/known_issues.rst

```diff
@@ -182,6 +183,17 @@ ZBOSS alarms inaccurate
 
   **Workaround:** Use Zephyr alarms.
 
+nRF Desktop
+===========
+
+.. rst-class:: v1-3-0 v1-3-1 v1-4-0
+
+DESK-978: Direct advertising issues with SoftDevice Link Layer
+  The direct advertising (:option:`CONFIG_DESKTOP_BLE_DIRECT_ADV`) should not be used by the :ref:`nrf_desktop` application when SoftDevice Link Layer is in use, because that leads to reconnection problems.
```

Comment:
```suggestion
  Directed advertising (:option:`CONFIG_DESKTOP_BLE_DIRECT_ADV`) should not be used by the :ref:`nrf_desktop` application when the :ref:`nrfxlib:softdevice_controller` is in use, because that leads to reconnection problems.
```